src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@ public interface CinemaHallDao {
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    boolean update(CinemaHall cinemaHall);
```

Comment:
The return type of the `update` method should be `boolean` instead of `Boolean` to avoid unnecessary boxing.